src/runtime/virtcontainers/mount.go

```diff
@@ -355,7 +358,7 @@ func bindUnmountAllRootfs(ctx context.Context, sharedDir string, sandbox *Sandbo
 		if c.state.Fstype == "" {
```

Comment:
This check here should prevent bindUnmountContainerRootfs from being called. If it were being called, the bindUnmountContainerRootfs call would have returned an error which would have been propogated up the stack.
Are you seeing otherwise?

Reply:
Any comment on this @keloyang?

Reply:
@keloyang Can you check with @amshinde about this scenario. I think what would be nice is we have an example of failure in the commit message.